superset/jinja_context.py

```diff
@@ -486,6 +487,26 @@ def process_template(self, sql: str, **kwargs: Any) -> str:
 
 
 class JinjaTemplateProcessor(BaseTemplateProcessor):
+    def _parse_datetime(self, dttm: Optional[str]) -> Optional[datetime]:
+        """
+        Try to parse a datetime and default to None in the worst case.
+
+        Since this may have been rendered by different engines, the datetime may
+        vary slightly in format. We try to make it consistent, and if all else
+        fails, just return None.
+        """
+        if not dttm:
+            return None
+
+        dttm_format = "%Y-%m-%dT%H:%M:%S.%f"
```

Comment:
I think `dateutils.parser.parse` is likely the preferred method of converting a `str` to a `datetime` object.

Reply:
done